src/components/SelectionList/BaseSelectionList.js

```diff
@@ -9,22 +9,32 @@ import TextInput from '../TextInput';
 import ArrowKeyFocusManager from '../ArrowKeyFocusManager';
 import CONST from '../../CONST';
 import variables from '../../styles/variables';
-import {propTypes as selectionListRadioPropTypes, defaultProps as selectionListRadioDefaultProps} from './selectionListRadioPropTypes';
+import {propTypes as selectionListPropTypes, defaultProps as selectionListDefaultProps} from './selectionListPropTypes';
 import RadioListItem from './RadioListItem';
+import CheckboxListItem from './CheckboxListItem';
 import useKeyboardShortcut from '../../hooks/useKeyboardShortcut';
 import SafeAreaConsumer from '../SafeAreaConsumer';
 import withKeyboardState, {keyboardStatePropTypes} from '../withKeyboardState';
+import Checkbox from '../Checkbox';
+import PressableWithFeedback from '../Pressable/PressableWithFeedback';
+import FixedFooter from '../FixedFooter';
+import Button from '../Button';
+import useLocalize from '../../hooks/useLocalize';
 
 const propTypes = {
     ...keyboardStatePropTypes,
-    ...selectionListRadioPropTypes,
+    ...selectionListPropTypes,
 };
 
-function BaseSelectionListRadio(props) {
+function BaseSelectionList(props) {
```

Comment:
We should use prop destructuring now -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring

Reply:
Cool, forgot it was recently merged, nice!